public/app/features/alerting/unified/rule-editor/RuleEditor.tsx

```diff
@@ -80,20 +80,23 @@ function NewRuleEditor() {
   const isManualRestore = useManualRestore();
   const { type = '', identifier = '' } = useRuleEditorPathParams();
 
-  const entityName = RECORDING_TYPE.includes(type)
-    ? t('alerting.editor.recording-rule-entity-name', 'recording rule')
-    : t('alerting.editor.alerting-rule-entity-name', 'alert rule');
+  const isExisting = Boolean(identifier);
+  const isRecordingRule = RECORDING_TYPE.includes(type);
 
-  const actionName = Boolean(identifier)
-    ? t('alerting.editor.edit-action', 'Edit')
-    : t('alerting.editor.new-action', 'New');
+  const newText = isRecordingRule
+    ? t('alerting.editor.new-recording-rule', 'New recording rule')
+    : t('alerting.editor.new-alert-rule', 'New alert rule');
+
+  const editText = isRecordingRule
+    ? t('alerting.editor.edit-recording-rule', 'Edit recording rule')
+    : t('alerting.editor.edit-alert-rule', 'Edit alert rule');
```

Comment:
We should have separate translation strings for the different cases (as we're otherwise assuming that it's always going to be {verb} {entity}, which might not be the case in all languages.

But also I wasn't sure how to get to the case of "Edit" - I see the title on the page in that case